api/v1alpha5/conversion.go

```diff
@@ -30,6 +30,28 @@ var _ ctrlconversion.Convertible = &OpenStackCluster{}
 
 const trueString = "true"
 
+// legacyCalicoSecurityGroupRules are the legacy security group rules for calico
```

Comment:
v1alpha5 conversion is really best effort at this time. Let me know if we need to do more.